src/components/screen/Oauth/Requests/Requests.vue

```diff
@@ -123,6 +140,13 @@ export default {
         this.stopLoading();
       }
     },
+
+    notifyThatPopupWasClosed() {
```

Comment:
maybe rename to `notifyCanceled`?

Reply:
I guess this is strange in terms of English, because in this case it will looks like notify cancelled (in otner words, if user cancels the notify). But we have popup closed. Maybe something more abstract but more correct like `notifyProcessCancelled`?

Reply:
yes, `notifyProcessCancelled` is better

Reply:
handleClose, because today we notify, tommorow we'll have other logic